Samples~/LootLockerExamples/Scenes/7 - PlayerFiles.unity.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: 5812616765f7593429400cc9024e3e43
```

Comment:
I ran this in my "test samples" ci test and this gives a warning because you've copied the meta from WhiteLabel... meta file so it has the same GUID, please fix :)